xonsh/commands_cache.py

```diff
@@ -166,10 +166,13 @@ def _update_paths_cache(self, paths: tp.Sequence[str]) -> bool:
                 self.cache_file.unlink(missing_ok=True)
 
         updated = False
+        is_darwin = os.uname().sysname.lower() == "darwin"
```

Comment:
the xonsh.platform has already similar functions. Lets reuse that.

Reply:
done